core/encodings.js

```diff
@@ -34,7 +34,10 @@ export const encodings = {
     pseudoEncodingCompressLevel9: -247,
     pseudoEncodingCompressLevel0: -256,
     pseudoEncodingVMwareCursor: 0x574d5664,
-    pseudoEncodingExtendedClipboard: 0xc0a1e5ce
+    pseudoEncodingExtendedClipboard: 0xc0a1e5ce,
+    pseudoEncodingRichCursor: 0xffffff11,
```

Comment:
This is just a different way of writing `-239`, so this is a duplicate of `pseudoEncodingCursor`.

Reply:
Fixed in e34a1c2